notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -944,7 +892,7 @@
       "source": [
         "## Customize TabNet HyperparameterTuningJob configuration and create pipeline\n",
         "\n",
-        "To get the best set of hyperparameters on your dataset, it is recommended to run a HyperparameterTuningJob.\n",
+        "To get the best set of hyperparameters on your dataset, it's recommended to run a HyperparameterTuningJob.\n",
```

Comment:
- change: "it's recommended to run a HyperparameterTuningJob"
- to: it's recommended that you run a HyperparameterTuningJob